src/js/common/components/CopyTextToClipboardButton/CopyTextToClipboardButton.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+import { IconButton, Tooltip } from '@material-ui/core';
+import { Done, FileCopyOutlined } from '@material-ui/icons';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+
+const CopyTextToClipboardButton = ({ textToCopy }) => {
+  const [isShowingSuccessTooltip, setIsShowingSuccessTooltip] = useState(false);
+
+  const { t } = useTranslation('common');
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(textToCopy);
+    setIsShowingSuccessTooltip(true);
+    setTimeout(() => {
+      setIsShowingSuccessTooltip(false);
+    }, 2000);
+  };
+
+  return (
+    <Tooltip open={isShowingSuccessTooltip} title={t('copiedToClipboard')} arrow placement='top'>
+      <IconButton
+        color={isShowingSuccessTooltip ? 'secondary' : 'default'}
+        onClick={copyToClipboard}
```

Comment:
I think that this button should be disabled when `isShowingSuccessTooltip === true`. If you use the disabled prop the button may become gray, so you can pass null to onClick or add a class with `pointer-events: none`.

Other alternative is to clear the timeout every time the user clicks the button, to do this you will need to do something like this:

```javascript
const timer = useRef(0)

const handleCopy = () => {
  clearTimeout(timer.current)
  // .........
  timer.current = setTimeout(...)
}
```